working/augmentation-libraries/feature-specification.md

```diff
@@ -1,7 +1,7 @@
 # Augmentation Libraries
 
 Author: rnystrom@google.com, jakemac@google.com
-Version: 1.10 (see [Changelog](#Changelog) at end)
+Version: 1.11 (see [Changelog](#Changelog) at end)
```

Comment:
We also talked about extending the language to allow abstract member declarations inside non-abstract classes (providing that they get augmented with a body before the class is compiled).

I'm not sure where that should go in here, but probably somewhere.

Reply:
I did that in one of the other PRs :)